functions/utils/audio.py

```diff
@@ -1,38 +1,71 @@
+import wave
 from pathlib import Path
 
-import soundfile as sf
-
 
 def get_sample_rate(file: Path) -> int:
-    """Gets the current sample rate of the given audio file.
+    """Gets the current sample rate of the given wav file.
 
     Parameters:
         file: The path to the audio file.
 
     Returns:
         The sample rate of the given file.
     """
-    _, sample_rate = sf.read(file)
+    with open(file, "rb") as f:
+        wave_object = wave.open(f, "rb")
+        sample_rate = wave_object.getframerate()
+        wave_object.close()
+
     return sample_rate
 
 
 def resample(file: Path, destination: Path, sample_rate: int) -> None:
-    """Copies an audio file to the destination, with the given
+    """Copies a wav file to the destination, with the given
     sample rate.
 
     Parameters:
         file (Path): The path of the file to resample
         destination (Path): The destination at which to create the resampled file
         sample_rate (int): The sample rate for the resampled audio.
     """
-    data, _ = sf.read(file)
-    sf.write(destination, data, sample_rate)
+    with open(file, "rb") as f:
+        wave_object = wave.open(f, "rb")
+        params = wave_object.getparams()
+        frames = wave_object.readframes(wave_object.getnframes())
+        wave_object.close()
+
+    with open(destination, "wb") as f:
+        result = wave.open(f, "wb")
+        result.setparams(params)
+        result.setframerate(sample_rate)
+        result.writeframes(frames)
+        result.close()
+
 
+def cut(file: Path, destination: Path, start_ms: int, stop_ms: int) -> None:
+    """Creates a new wav file at the destination, restricted to the given start
+    and stop times.
+
+    Parameters:
+        file (Path): The path of the file to resample.
+        destination (Path): The destination at which to create the resampled file.
+        start_ms (int): The start time in milliseconds to record from.
+        stop_ms (int):  The stop time in milliseconds to record to.
+    """
+    with open(file, "rb") as f:
+        wave_object = wave.open(f, "rb")
+        params = wave_object.getparams()
+        frames = wave_object.readframes(wave_object.getnframes())
+        width = wave_object.getsampwidth()
+        sample_rate = wave_object.getframerate()
+        wave_object.close()
 
-def cut(
-    file: Path, destination: Path, sample_rate: int, start_ms: int, stop_ms: int
-) -> None:
     start = round(sample_rate * start_ms / 1000)
     stop = round(sample_rate * stop_ms / 1000)
-    data, _ = sf.read(file, start=start, stop=stop)
-    sf.write(destination, data, sample_rate)
+
+    with open(destination, "wb") as f:
+        result = wave.open(f, "wb")
```

Comment:
I don't feel great about `result` as a variable name here.

Reply:
I made it a bit more palateable for both cut and resample. (destination_wave)?